projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.ts

```diff
@@ -1404,7 +1404,7 @@ export class IgxPivotGridComponent extends IgxGridBaseDirective implements OnIni
     /**
      * @hidden @internal
      */
-    public override endEdit(_commit = true, _event?: Event) {
+    public override endEdit(_commit = true, _event?: Event): any {
```

Comment:
please avoid 'any' as a type ... the return type for this method should be boolean